src/agent.ts

```diff
@@ -0,0 +1,58 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
```

Comment:
```suggestion
import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
```

Unused imports can be removed.